pkg/quota/evaluator/core/pods.go

```diff
@@ -101,6 +104,16 @@ func PodConstraintsFunc(required []api.ResourceName, object runtime.Object) erro
 			missingSet.Insert(difference.List()...)
 		}
 	}
+	for i := range pod.Spec.InitContainers {
+		requests := pod.Spec.InitContainers[i].Resources.Requests
+		limits := pod.Spec.InitContainers[i].Resources.Limits
+		containerUsage := podUsageHelper(requests, limits)
```

Comment:
can we have a function that does:

```enforceContainerConstraints(container *Container, required, missing set)```

so we can just call that with either an init or a regular container and not duplicate code?